content/en/agent/faq/windows-agent-ddagent-user.md

```diff
@@ -0,0 +1,42 @@
+---
+title: ddagentuser with the Windows Agent 
+kind: faq
+disable_toc: true
+---
+
+**Starting with release `6.11`, the Windows Agent main components (i.e. Core, APM/Trace, and Process) run under the `ddagentuser` account, created at install time, instead of running on prior version under the `LOCAL_SYSTEM` account.**
+
+The user `ddagentuser` is created at install time for the Datadog Windows Agent. When installed on an Active Directory server, the username and password must be provided to the installer. The new user is a non-privileged user. It gains the following rights during installation:
+
+* It can start and stop the APM and Process Agent
+* It becomes a member of the “Performance Monitor Users” group
+* Necessary to access WMI information
```

Comment:
These two lines (`necessary....`) might be indented?  They indicate _why_ we add the user to the Performance Monitor Users group